src/transpiler.zig

```diff
@@ -992,6 +992,8 @@ pub const Transpiler = struct {
 
         dont_bundle_twice: bool = false,
         allow_commonjs: bool = false,
+        /// `"type"` from `package.json`
```

Comment:
```suggestion
        /// `"type"` from `package.json`. Used to make sure the parser defaults to CommonJS or ESM based on what the package.json says, when it doesn't otherwise know from reading the source code.
```